tests/deploy/utils.py

```diff
@@ -0,0 +1,40 @@
+import os
+import time
+
+from fastapi.testclient import TestClient
+
+from ragna.assistants import RagnaDemoAssistant
+from ragna.core._utils import default_user
+
+
+class TestAssistant(RagnaDemoAssistant):
+    def answer(self, prompt, sources, *, multiple_answer_chunks: bool):
```

Comment:
The pydantic error that happens as part of the error log in is

```
pydantic_core._pydantic_core.ValidationError: 1 validation error for 1d49d534-1b0e-4d37-bbf7-3d922f898376
multiple_answer_chunks
  Field required [type=missing, input_value={'user': '', 'chat_id': U...at 05/22/2024 10:21 AM'}, input_type=dict]
    For further information visit https://errors.pydantic.dev/2.7/v/missing
```

It complains that this assistant requires the `multiple_answer_chunks` parameter since it has no default, but we didn't pass it. Unfortunately, there is no way to do this from the web UI at the moment. See https://ragna.chat/en/latest/generated/tutorials/gallery_custom_components/#web-ui_1 for details.

I'm ok with re-using this `TestAssistant` for the API and UI tests. But to make it work for the UI, we need to give `multiple_answer_chunk: bool = True` a default value. Just add a comment why the default value is there so we can remove it later on when we no longer need it.